content/docs/guides/10-minute-tutorial.md

```diff
@@ -879,6 +885,7 @@ package hellocucumber;
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.When;
 import cucumber.api.java.en.Then;
+
```

Comment:
This new line was generated by the maven archetype command. :man_shrugging: 
Hence the line number changes in the stacktraces.

Reply:
Thats okay. IIRC IDEA auto formats imports to separate static and non static imports.